plugins/ctx.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: ctx
+spec:
+  homepage: https://github.com/ahmetb/kubectx
+  shortDescription: Switch between clusters in your kubeconfig
+  version: v0.7.1
+  description: |
+    Also known as "kubectx", a utility to switch between context entries in
+    your kubeconfig file effectively.
+  caveats: |
+    If fzf is installed on your machine, you can interactively choose
```

Comment:
It would make sense to include a link to the fzf repo.

Reply:
I think fzf is easily findable and makes the text longer, so we should leave it out. I'm similarly thinking all these plugins listing USAGE strings in  their caveats/description. Similarly not ideal.